org.hl7.fhir.validation/src/main/java/org/hl7/fhir/validation/instance/InstanceValidator.java

```diff
@@ -4528,7 +4528,16 @@ public void checkChildByDefinition(ValidatorHostContext hostContext, List<Valida
     String thisExtension = null;
     boolean checkDisplay = true;
 
-    checkInvariants(hostContext, errors, profile, typeDefn != null ? typeDefn : checkDefn, resource, ei.getElement(), localStack, true);
+    // Lloyd: Changed onlyNonInherited to false because inherited invariants can be overridden and it's important to check both.  
+    // Caching logic should make checking inherited constraints more efficient now.
+    // Added choice around checkInvariants because azatadha/example-params-1 test was treating the '%resource' in the bdl-3 constraint for 
+    // evaluating the Bundle rule as Parameters, not Bundle
+    SpecialElement special = ei.getElement().getSpecial();
```

Comment:
So, 
1. Please put this in the RELEASE_NOTES.md file, and not here. 
2. If you want to keep a comment here, which would be great, please reformat it just to say what the code does, no need for your name or added verbs describing the work process